packages/ubus-lime-utils/Makefile

```diff
@@ -13,7 +13,7 @@ define Package/$(PKG_NAME)
   MAINTAINER:=Marcos Gutierrez <gmarcos87@gmail.com>
   SUBMENU:=3. Applications
   TITLE:=LIbremesh ubus utils module
-  DEPENDS:= +lua +libubox-lua +libubus-lua +libuci +cgi-io +rpcd-mod-file
+  DEPENDS:= +lua +libubox-lua +libubus-lua +libuci +cgi-io +rpcd-mod-file +lime-system
```

Comment:
Are you really really sure `lime-system` is needed?

Reply:
https://github.com/libremesh/lime-packages/issues/766#issuecomment-702940526
and
https://github.com/libremesh/lime-packages/blob/5403093ba9fe47dedef1345df5d08cf9f1a34072/packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils#L12-L14